tests/units/units.cpp

```diff
@@ -665,6 +665,29 @@ TEST(Units, compareMultiplierStandardUnit)
     EXPECT_NEAR(500.0, libcellml::Units::scalingFactor(u2, u1), 1e-12);
 }
 
+TEST(Units, compareMultiplierFirstParameterNullUnit)
```

Comment:
Rename the test to something like `compareScalingFactorWithNullptrAsFirstParameter`?